### ISSUE TITLE
Missing label or miss-labeled buttons

### ISSUE THREAD

**Issue by connorscarolyns** [cds-snc/platform-forms-client#5183]
## From Fable testing

On each screen of the form, the Continue, back, and Save to Device buttons are not labelled. For clarification, I would expect all buttons to be clearly labelled. I will not know what these buttons are otherwise. 

Although it is possible to proceed through the test survey, there are a series of unlabeled buttons at the bottom of each succeeding page, which allow you to proceed, or perhaps cancel. This is on the disclosure page where you can accept or decline to proceed. On the next set of pages there are buttons which I believe are used to go back, to go to the next page in the form, and to save the document on ones device. These buttons simply have no labeling. I have not experienced any labels on these particular navigation buttons in this entire form.

I tried to use Voice Control's "click" command by saying "click resume form," however nothing happened. I used my "names overlay" to determine the label of the "resume form" clickable element and it appears to be "resume form continue filling..." Ideally it would just be labeled "resume form."

On the next page, after selecting "resume form" using the "numbers overlay," I ran into the same difficulty with the button called "resume" because the label for that button is "Resume upload your saved" I appreciate that this could be helpful for users of other types of assistive technology because the longer label gives more information. In that case, it would be great for Voice Control users, if these buttons could have alternative labels, not only one label per button. The alternative label could be "resume" and so Voice Control users would easily be able to use the "click" command.

**Comment by anikbrazeau** [comment]
Related #6968

**Comment by anikbrazeau** [comment]
Related #4109 #5384

**Comment by anikbrazeau** [comment]
Additionally: 

https://cds-snc.freshdesk.com/a/tickets/29577
> Button mouseovers need to be added. An Accessibility feature for buttons is that a mouseover needs to be provided to tell the user what the button does. For example, a “Continue” button should provide a mouseover of “Continues to the next page”. All buttons should have mouseover text on both the English and French pages.
